operator/controllers/setup.go

```diff
@@ -72,6 +71,14 @@ func (r *KymaReconciler) SetupWithManager(mgr ctrl.Manager, options controller.O
 	return nil
 }
 
+func deployChartConfig(skrChartConfig *SkrChartConfig) deploy.ManagerConfig {
```

Comment:
This function is not necessary, you just create a struct, so i think you can simply create it in setupWithManager.